grandcypher/__init__.py

```diff
@@ -754,6 +761,15 @@ def returns(self, ignore_limit=False):
             for key, values in results.items()
             if self._alias2entity.get(key, key) in self._return_requests
         }
+        # HACK: convert to [None] if edge is None
```

Comment:
this is clever!!

Reply:
this is only necessary for DiGraph input, so I’m sure there’s a cleaner way to do it🙂